src/RestApi/StoreApi/Schemas/OrderSchema.php

```diff
@@ -0,0 +1,625 @@
+				'company'    => $object->get_billing_company(),
+				'address_1'  => $object->get_billing_address_1(),
+				'address_2'  => $object->get_billing_address_2(),
+				'city'       => $object->get_billing_city(),
+				'state'      => $object->get_billing_state(),
+				'postcode'   => $object->get_billing_postcode(),
+				'country'    => $object->get_billing_country(),
+				'email'      => $object->get_billing_email(),
+				'phone'      => $object->get_billing_phone(),
+			],
+			'shipping_address'   => [
+				'first_name' => $object->get_shipping_first_name(),
+				'last_name'  => $object->get_shipping_last_name(),
+				'company'    => $object->get_shipping_company(),
+				'address_1'  => $object->get_shipping_address_1(),
+				'address_2'  => $object->get_shipping_address_2(),
+				'city'       => $object->get_shipping_city(),
+				'state'      => $object->get_shipping_state(),
+				'postcode'   => $object->get_shipping_postcode(),
+				'country'    => $object->get_shipping_country(),
+			],
+			'items'              => array_values( array_map( [ $order_item_schema, 'get_item_response' ], $object->get_items( 'line_item' ) ) ),
+			'totals'             => array_merge(
+				$this->get_store_currency_response(),
+				[
+					'total_items'        => $this->prepare_money_response( $object->get_subtotal(), wc_get_price_decimals() ),
+					'total_items_tax'    => $this->prepare_money_response( $this->get_subtotal_tax( $object ), wc_get_price_decimals() ),
+					'total_fees'         => $this->prepare_money_response( $this->get_fee_total( $object ), wc_get_price_decimals() ),
+					'total_fees_tax'     => $this->prepare_money_response( $this->get_fee_tax( $object ), wc_get_price_decimals() ),
+					'total_discount'     => $this->prepare_money_response( $object->get_discount_total(), wc_get_price_decimals() ),
+					'total_discount_tax' => $this->prepare_money_response( $object->get_discount_tax(), wc_get_price_decimals() ),
+					'total_shipping'     => $this->prepare_money_response( $object->get_shipping_total(), wc_get_price_decimals() ),
+					'total_shipping_tax' => $this->prepare_money_response( $object->get_shipping_tax(), wc_get_price_decimals() ),
+					'total_price'        => $this->prepare_money_response( $object->get_total(), wc_get_price_decimals() ),
+					'total_tax'          => $this->prepare_money_response( $object->get_total_tax(), wc_get_price_decimals() ),
+					'tax_lines'          => $this->get_tax_lines( $object ),
+				]
+			),
+		];
+	}
+
+	/**
+	 * Removes the wc- prefix from order statuses.
+	 *
+	 * @param string $status Status from the order.
+	 * @return string
+	 */
+	protected function remove_status_prefix( $status ) {
+		return 'wc-' === substr( $status, 0, 3 ) ? substr( $status, 3 ) : $status;
+	}
+
+	/**
+	 * Get event dates from an order, formatting both local and GMT values.
+	 *
+	 * @param \WC_Order $object Order class instance.
+	 * @return array
+	 */
+	protected function get_events( $object ) {
```

Comment:
We should be able to typehint this here right?  Also any reason the variable name couldn't be `$order` instead of `$object`?